homeworks/roman.pavliuk_RomanPvlk/PokemonProject/style.css

```diff
@@ -0,0 +1,46 @@
+body {
+  margin: 0 auto;
+}
+
+.about_section {
+  max-width: 1175px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+}
+
+.about_section__image {
+  width: 532px;
+  height: 297px;
+}
+
+.about_section__text {
+  width: 575px;
+}
+
+.about_section__text h1 {
+  font-family: Roboto, serif;
```

Comment:
[Roboto is a sans-serif font](https://en.wikipedia.org/wiki/Roboto#:~:text=Roboto%20(%2Fro%CA%8A%CB%88b,4.0%20%22Ice%20Cream%20Sandwich%22.), so if you like to state here a generic default, please do it right 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/8546f740-bc8a-47bf-adbd-e81e81e66435)